ibmjava/8/jre/ubi-min/Dockerfile

```diff
@@ -21,33 +21,34 @@
 
 FROM registry.access.redhat.com/ubi7/ubi-minimal
 
-MAINTAINER Dinakar Guniguntala <dinakar.g@in.ibm.com> (@dinogun)
 
-RUN microdnf install openssl wget ca-certificates gzip tar && \
-    microdnf update; microdnf clean all
+RUN microdnf install openssl wget ca-certificates gzip tar \
+    && microdnf update; microdnf clean all
 
+LABEL author="Dinakar Guniguntala <dinakar.g@in.ibm.com> (@dinogun)" \
```

Comment:
Can you this LABEL to `org.opencontainers.image.authors` as this is the recommended convention per docker. Also please your name to the authors as well.